docs/source/keywords.rst

```diff
@@ -494,77 +494,77 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
 AMD GPUs
 ~~~~~~~~
 
+The Kokkos naming convention is to aggregate AMD\_ and the architecture flag.
+
+``Kokkos_ARCH_AMD_<ARCHITECTURE_FLAG>``
+
+If the HIP backend is enabled and no AMD GPU architecture is specified,
+Kokkos will attempt to autodetect the architecture flag at configuration time.
+
 .. list-table::
-    :widths: 25 65 10
+    :widths: 30 15 25 30
     :header-rows: 1
     :align: left
 
     * - **AMD GPUs**
-      - Description/info
-      - Default
+      - Architecture flags
+      - Models
+      - Notes
+
+    * * ``Kokkos_ARCH_AMD_GFX942``
+      * GFX942
+      * MI300A, MI300X
+      * (since Kokkos 4.2)
+
+    * * ``Kokkos_ARCH_AMD_GFX940``
+      * GFX940
+      * MI300A (A0 part)
```

Comment:
Is there a description anywhere of what A0 refers to/mean?

Reply:
(not asking you to add a link to the page, just wondering how to make sense of that info)

Reply:
May be we could say "MI300 (pre-production)"

Reply:
Done